### ISSUE TITLE
Card mod filter for specific event

### ISSUE THREAD

**Issue by cofw2005** [FamousWolf/week-planner-card#141]
Excuse me! Could you please kindly help? I would like to set the event background card based on the name of the event.
For example, this is the event list:
```
calendar.my_calender:
  events:
    - start: "2024-09-16T08:00:00+02:00"
      end: "2024-09-16T09:30:00+02:00"
      summary: Geographie
      location: 2. OG 210
    - start: "2024-09-16T09:45:00+02:00"
      end: "2024-09-16T11:15:00+02:00"
      summary: Sport
      location: Turnhalle links
    - start: "2024-09-16T11:30:00+02:00"
      end: "2024-09-16T12:15:00+02:00"
      summary: Informatik
      location: 1. OG 105
```
For example, if I want to apply background color: red for Geographie, green for Sport. How can I achieve this with the card mod? I tried with | selectattr() filter but didn't get success. Please kindly help.
Thanks very much!

**Comment by FamousWolf** [comment]
Unfortunately there's no selector based on the text in elements in CSS (yet). I could add the summary as a data attribute to the event div. Then you could use something like `div.event[data-summary="Geographie"] { ... }` to style based on the summary.

**Comment by FamousWolf** [comment]
This has been added to the dev branch and will be included in the 1.9.0 release.

I've added data attributes for summary, location, start hour, start minute, end hour and end minute, so any of these can be used for custom styling.
